src/main/scala/advancepassengerinfo/importer/DqApiFeed.scala

```diff
@@ -26,9 +26,12 @@ case class DqApiFeedImpl(fileNamesProvider: FileNames,
       .unfoldAsync((lastFileName, List[String]())) { case (lastFileName, lastFiles) =>
         markerAndNextFileNames(lastFileName).map {
           case (nextFetch, newFiles) => Option((nextFetch, newFiles), (lastFileName, lastFiles))
+        }.recover {
```

Comment:
Looking at the tests I think we are missing a case for a failure here, so would be good to test that one to see what actually happens. It might be that in the recover that we need to return the previous values, ie `lastFileName` & `lastFiles`. I'm struggling to remember exactly how `unfold` works as this is the only place I've ever used it